src/Integration.Vsix/Analysis/DisableRuleCommand.cs

```diff
@@ -149,22 +150,18 @@ private void Execute(object sender, EventArgs e)
         /// 1) there is only one selected error in the error list and
         /// 2) it is a SonarLint error
         /// </summary>
-        internal /* for testing */ static  bool TryGetErrorCodeSync(IErrorList errorList, out string errorCode)
```

Comment:
The old comment was inaccurate: there was a third condition: "3) the language supports disabling".
The comment is now accurate, as the enable/disable check is done elsewhere.